dolphinscheduler-master/src/main/bin/start.sh

```diff
@@ -19,7 +19,7 @@
 BIN_DIR=$(dirname $0)
 DOLPHINSCHEDULER_HOME=${DOLPHINSCHEDULER_HOME:-$(cd $BIN_DIR/..; pwd)}
 
-source "$BIN_DIR/dolphinscheduler_env.sh"
+source "$BIN_DIR/../conf/dolphinscheduler_env.sh"
```

Comment:
What about just 

```suggestion
source "$DOLPHINSCHEDULER_HOME/conf/dolphinscheduler_env.sh"
```

Reply:
Will change it

Reply:
Done and could you please take a look again? BTW, why we should add file `dolphinscheduler_env.sh` in all servers instead of using the file in the path `bin/env/dolphinscheduler_env.sh`?

Reply:
> BTW, why we should add file `dolphinscheduler_env.sh` in all servers instead of using the file in the path `bin/env/dolphinscheduler_env.sh`?

See my comment https://github.com/apache/dolphinscheduler/issues/9675#issuecomment-1107996038 . The short answer is that `bin/env/dolphinscheduler_env.sh` is only available when you use `install.sh` to install all components in a single machine. If you install each component in a separate machine, `bin/env/dolphinscheduler_env.sh` is not available there, also cc @SbloodyS